[BUG] `kurt` and `skew` returning incorrect results

**Describe the bug**
`kurt` and `skew` seems to be returning incorrect results for some specific cases. 

Following are the pytests failing:
```
FAILED python/cudf/cudf/tests/test_reductions.py::test_reductions_axis_none_warning[kurt]
FAILED python/cudf/cudf/tests/test_reductions.py::test_reductions_axis_none_warning[kurtosis]
FAILED python/cudf/cudf/tests/test_reductions.py::test_reductions_axis_none_warning[skew]
```
```python
(Pdb) df
   a   b
0  1  10
1  2   2
2  3   3
(Pdb) df.kurt(axis=None)
nan
(Pdb) df.to_pandas().kurt(axis=None)
4.965499170233208

```

--- a/python/cudf/cudf/core/dataframe.py
+++ b/python/cudf/cudf/core/dataframe.py
@@ -57,6 +57,7 @@ from cudf.api.types import (
     is_string_dtype,
 )
 from cudf.core import column, df_protocol, indexing_utils, reshape
+from cudf.core._compat import PANDAS_GE_200
 from cudf.core.abc import Serializable
 from cudf.core.column import (
     CategoricalColumn,
@@ -95,11 +96,8 @@ from cudf.utils.dtypes import (
     min_scalar_type,
     numeric_normalize_types,
 )
-
 from cudf.utils.nvtx_annotation import _cudf_nvtx_annotate
 from cudf.utils.utils import GetAttrGetItemMixin, _external_only_api
-from cudf.core._compat import PANDAS_GE_200
-
 
 _cupy_nan_methods_map = {
     "min": "nanmin",
@@ -6112,8 +6110,13 @@ class DataFrame(IndexedFrame, Serializable, GetAttrGetItemMixin):
                     if axis == 0
                     else source.index
                 )
-
         if axis in {0, 2}:
+            if axis == 2 and op in ("kurtosis", "kurt", "skew"):
+                # TODO: concat + op can probably be done in the general case
+                # for axis == 2.
+                return getattr(concat_columns(source._data.columns), op)(
+                    **kwargs
+                )
             try:
                 result = [
                     getattr(source._data[col], op)(**kwargs)

